drivers/mm/Kconfig

```diff
@@ -17,6 +17,14 @@ config MM_DRV_PAGE_SIZE
 	help
 	  Size of memory pages.
 
+config MM_DRV_POWER_OFF_UNUSED_RAM
+	bool "Power off unused RAM"
+	help
+	  Powers off banks of RAM that were not allocated
+	  for use during linkage. Disable this option if
+	  dynamically managing memory, such as by using
+	  a heap allocator.
```

Comment:
default y please 

This is a standard way of memory managing in ADSP Meteorlake

Reply:
Keeping the default to `y` will break newlib at least - maybe more stuff. I don't think that changing these unrelated places (to add CONFIG_=`n`) due this behaviour is the right approach. I'd say that those using it should enable it (and take care of preventing issues). Maybe the approach used in this patch is the wrong one with regard to these issues - thoughts?

Reply:
AFAIK adsp board is used by SOF only, is it not?

Reply:
> AFAIK adsp board is used by SOF only, is it not?

yes, but we need to test all zephyr features without SOF and some might require this to be enabled/disabled. Safer to set this to y by the user to avoid any other issues.

Reply:
So why not leave this patch as is (or change the name and scope of the config) and enable the new config on SOF only. Less breakage overall...

Reply:
> This is a standard way of memory managing in ADSP Meteorlake

right, but in Zephyr we do not enable things by default that might conflict with tests and other areas of the OS, so this should be enabled based on need.

Reply:
Ok, will put a PR in sof than